Cargo.toml

```diff
@@ -83,33 +84,33 @@ all-extensions = [
 ]
 
 # Features to enable individual X11 extensions
-composite = ["xfixes"]
-damage = ["xfixes"]
-dpms = []
-dri2 = []
-dri3 = []
-glx = []
-present = ["randr", "xfixes", "sync"]
-randr = ["render"]
-record = []
-render = []
-res = []
-screensaver = []
-shape = []
-shm = []
-sync = []
-xevie = []
-xf86dri = []
-xf86vidmode = []
-xfixes = ["render", "shape"]
-xinerama = []
-xinput = ["xfixes"]
-xkb = []
-xprint = []
-xselinux = []
-xtest = []
-xv = ["shm"]
-xvmc = ["xv"]
+composite = ["x11rb-protocol/composite", "xfixes"]
```

Comment:
Did you forget to change `xfixes` to `x11rb-protocol/xfixes` or did I miss something? Same for the lines below.

Reply:
Nope, not really.

Previously, the line was
```
composite = ["xfixes"]
```
This meant that enabling the `composite` feature also needs to enable the xfixes feature. This did not change. I only also enabled the `composite` feature in `x11rb-protocol`. Its `Cargo.toml` then does the same dance to also enable its `xfixes`-feature.

Both crates need the `composite -> xfixes` dependency because there is a `use super::xfixes;` in `composite.rs`.

Reply:
ooops, I see, I did not notice that the extension names where being repeated